dev-libs/rapidjson/rapidjson-1.1.0.recipe

```diff
@@ -0,0 +1,66 @@
+SUMMARY="A fast JSON parser and generator for C++ with both SAX/DOM style API"
+DESCRIPTION="RapidJSON is a JSON parser and generator for C++. \
+It was inspired by RapidXml. RapidJSON is small but complete. \
+RapidJSON is fast. Its performance can be comparable to strlen(). \
+RapidJSON is self-contained and header-only. RapidJSON is memory-friendly. \
+RapidJSON is Unicode-friendly."
+HOMEPAGE="https://github.com/Tencent/rapidjson"
+COPYRIGHT="2015 THL A29 Limited, a Tencent company
+	2015 Milo Yip"
+LICENSE="MIT
+	JSON"
+REVISION="1"
+SOURCE_URI="https://github.com/Tencent/rapidjson/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="bf7ced29704a1e696fbccf2a2b4ea068e7774fa37f6d7dd4039d0787f8bed98e"
+SOURCE_FILENAME="rapidjson-$portVersion.tar.gz"
+PATCHES="rapidjson-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	rapidjson$secondaryArchSuffix = $portVersion
+	lib:rapidjson$secondaryArchSuffix = $portVersion
```

Comment:
Does this package contain a library (`.so` file)? If not, you should merge the `_devel` package here and drop `lib:`.

Also, as this port uses [Semantic Versioning](https://semver.org/), you should use the string `compat >= ${portVersion%%.*}`.